src/style.css

```diff
@@ -0,0 +1,116 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf),
+    url(fonts/Avenir-Heavy.ttf),
+    url(fonts/Avenir.ttc);
+}
+
+.bigSearch {
+  width: 100%;
+  margin-top: 20px;
+  padding: 25px 5px 23px 62px;
+  font-weight: 300;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.bigButton {
+  display: inline-block;
+  width: 19px;
+  height: 19px;
+  padding: 0;
+  margin: 0;
+  background-color: transparent;
+  border: 0;
+  cursor: pointer;
+  position: relative;
+  top: 70px;
+  left: 28px;
+}
+
+.bigSearch::placeholder {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.bigSearch:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.bigSearch:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.smallSearch {
+  width: 100%;
+  margin: 20px 0;
+  padding: 12px 5px 10px 33px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+}
+
+.smallButton {
```

Comment:
create common class for forms, buttons & inputs, set all common properties to that class, and all different to their own